installer/manifests/keptn/values.yaml

```
@@ -14,11 +14,15 @@ mongo:
   host: mongodb:27017
   ## @param mongo.architecture
   architecture: standalone
+  updateStrategy:
+    ## @param mongo.updateStrategy.type
+    type: Recreate
   service:
     ## @param mongo.service.nameOverride
     nameOverride: 'mongo'
-    ## @param mongo.service.port
-    port: 27017
+    ports:
+      ## @param mongo.service.ports.mongodb
```

You could already add some small documentation

```suggestion
      ## @param mongo.service.ports.mongodb Port for MongoDB to listen at
```